Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Demonic Government

## Changelog:
- Added Demonic Government:
  - No vassal contracts, taxes and levies based on opinion.
  - Only rulers of the same religious family can inherit.
  - Dark Authority instead of Crown Authority:
    - Frenzy decays over time so to be efficient they should do their mechanics.
    - High Frenzy gives you movement speed, dread and tyranny bonuses.
  - Get Frenzy on executions.
  - Get Frenzy on torturing.
  - Attracts Demon Knights and Guests with high Skills.
- Assigned modifiers/effects to Demonic races: satyrs, observers, etc.

# Developer changelog:
- Added debug decisions to lose and give Dark Frenzy.

# Tasks:
- [x] Add debug decisions
- [x] Mention unique bonuses in government concept.


# How to test:
Test gameplay features, tell if they look balanced.

In the Debug mode, right-click Demonic government rulers to Give and Lose Dark Frenzy fast.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/129772245-54e7262d-ff92-43ed-b554-483e604d5042.png)

</details>

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/592/files?file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here are the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).
